scheduler/cups-driverd.cxx

```diff
@@ -2400,6 +2400,12 @@ load_ppds(const char *d,		/* I - Actual directory */
   */
 
   dinfoptr = (struct stat *)malloc(sizeof(struct stat));
+  if (!dinfoptr)
+  {
+    fputs("ERROR: [cups-driverd] Unable to allocate memory for directory info!\n",
```

Comment:
Don't need the exclamation mark - a period is fine here. Also, if we want to be consistent with some of the existing code you can do the assignment and test in the "if", e.g.:

```
if ((dinfoptr = ...) == NULL)
{
  fputs(...);
  exit(1);
}
```